### PR TITLE
[12-hsinchu] Create December scisprint page

### PR DESCRIPTION
**Pull request by YenPeiChen07**
@yungyuc, I have created a page for December sprint. Also added some information about sprint training room. 
Please take a look and give me some feedback.

### CODE REVIEW

File: content/pages/sprint/2022/12-hsinchu.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+should be added in the `sprint note <#sprint-note>`__ too.  Please `contact us
+<#contact-us>`__ if you have any questions.
+
+Scisprint is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+In this sprint, we provide a `Sprint Training room <#Sprint Training room>`__
```
Comment (yungyuc) [inline content/pages/sprint/2022/12-hsinchu.rst:26]: Could you double check whether or not the target works?  IIUC rst connects the words using hyphen (`-`).

Please also correct the capitalization.  If the first two words are capitalized, the third one should be as well.

File: content/pages/sprint/2022/12-hsinchu.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+programming.
+
+In this sprint, we provide a `Sprint Training room <#Sprint Training room>`__ 
+for the newcomers/less experienced developers to get prepared to contribute in the open-source projects.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+
+Sprint Training room
```
Comment (yungyuc) [inline content/pages/sprint/2022/12-hsinchu.rst:33]: It's great to have this new section.

**Comment by yungyuc** [comment]
> @yungyuc, I have created a page for December sprint. Also added some information about sprint training room.
> Please take a look and give me some feedback.

Look good to me except some minor things.  But we will need help to fix the CI.  Cue @chestercheng .

**Comment by YenPeiChen07** [comment]
@yungyuc, thanks for the comments. I have updated some details

**Comment by yungyuc** [comment]
LGTM.  I think the PR can leave the draft mode.

**Comment by yungyuc** [comment]
@YenPeiChen07 Could you make a new commit to trigger the build again?  The CI was fixed by @chestercheng with #153 .

**Comment by YenPeiChen07** [comment]
I have updated it to the newest develop branch.

File: content/pages/sprint/2022/12-hsinchu.rst [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Venue
+-----
+
+`國立清華大學第三綜合大樓 A區 5樓 <https://goo.gl/maps/EH2wWtkLQ8qLWd669>`__. Please see the `sprint
+note <#sprint-note>`__ for the access of the venue.
+
+.. raw:: html
+
+  <div style="overflow:hidden; padding-bottom:56.25%; position:relative; height:0;">
```
Comment (yungyuc) [inline content/pages/sprint/2022/12-hsinchu.rst:56]: Could you make it as wide as the page?  Currently it's narrow:

image

Perhaps you can take a look at https://sciwork.dev/sprint/2022/07-taipei .  The code is at https://github.com/sciwork/swportal/blob/f68ca3dcc17931c97459273f3ba994f898853334/content/pages/sprint/2022/07-taipei.rst?plain=1#L41

**Review by yungyuc — CHANGES_REQUESTED** [review]
Thanks a lot.  I think we can use a minor change before merging.

File: content/pages/sprint/2022/12-hsinchu.rst (hunk 3 above)
Comment (YenPeiChen07) [reply content/pages/sprint/2022/12-hsinchu.rst:56]: Sure. I could fix it.

**Comment by yungyuc** [comment]
It's good.  Thanks a lot.

**Pull request merged**